indra/assemblers/pybel/assembler.py

```diff
@@ -653,6 +658,39 @@ def _get_evidence(evidence):
             annotations[key] = {v: True for v in value}
         else:
             annotations[key] = {value: True}
+
+    if evidence.context:
+        context_annotations = []
+
+        attrs = ['location', 'cell_line', 'cell_type', 'organ',
+                 'disease', 'species']
+        for attr_key in attrs:
+            attr_val = evidence.context.__dict__.get(attr_key)
+            if attr_val:
+                for key, value in attr_val.db_refs.items():
+                    context_annotations.append((attr_key, f'{key}:{value}'))
+
+        # Add the annotations
+        for key, value in context_annotations:
+            annotations[key] = {value: True}
+
+    if evidence.annotations:
+        ignore_keys = {"agents", "raw_grounding", "bel", "prior_uuids",
+                       "provenance", "found_by", "text_refs", "obj_polarity",
+                       "obj_adjectives", "ANNOTATOR", "obj_adjectives",
+                       "SEQUENCE", "subj_context", "obj_context", ""}
```

Comment:
I'm not sure that adding any annotation we come across other than ones listed here is a good idea. We might consider flipping the logic here and allowing the parameterization of which annotations keys to include explicitly while ignoring all others.

Reply:
Yes you are right. Controlling what comes in makes sense.

Reply:
I changed the implementation to add an argument to the PybelAssembler called `annotations_to_include` that allows controlling externally which annotation entries to include in the PyBEL graph.